org.eclipse.scanning.api/src/org/eclipse/scanning/api/annotation/ui/FieldValue.java

```diff
@@ -300,9 +300,9 @@ public static boolean isModelField(Object model, String name) throws NoSuchField
 	 * @return fields old value, or null
 	 */
 	private static Object set(Object model, String name, Object value)throws IllegalAccessException, IllegalArgumentException, InvocationTargetException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.lang.IllegalArgumentException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)